### DISCUSSION TITLE
Using FA icons in menu items

### DISCUSSION THREAD

**Discussion by chopinesque** [bootscore/bootscore#894] [Q&A] [answered]
Would this be possible? I have used it in a different bootstrap theme
![iconsboot](https://github.com/user-attachments/assets/8289eeb4-86a9-4c8f-b61d-f89a11c0e929)

**Comment by crftwrk** [comment, marked as answer]
Sure, just add the icon HTML to the menu item:

<img width="438" alt="icon" src="https://github.com/user-attachments/assets/b710b048-228a-4c89-b961-f39fcb42aed5" />

**Comment by chopinesque** [comment]
Lol, that simple! I tried though to create sublevels in the menu in order to replicate the menu in the image (I guess what I have in the image is some type of mega menu?) but the deeper menu levels did not appear on the menu.

**Reply by crftwrk** [reply]
Bootstrap does not support more than 2 levels in dropdowns. If you want to have deeper levels for the menu, you have to modify the nav-walker. Follow this guide: https://github.com/orgs/bootscore/discussions/853

**Reply by crftwrk** [reply]
Please mark answer as resolved
